upload-api/test/helpers/ucan.js

```diff
@@ -266,7 +266,11 @@ export async function executionContextToUcantoTestServerContext (t) {
         t.fail(error.message);
       },
     },
-    maxUploadSize: 5_000_000_000,
+    // The largest object that can be uploaded in a single PUT is 5 GB
+    // https://aws.amazon.com/s3/faqs/
+    // However, this would allow for only 5 shards per Filecoin sector (if all
+    // 5GB). 4GB would allow a minimum 7 shards per sector.
+    maxUploadSize: 4_261_412_864,
```

Comment:
w00t 

I would add to the comment how to arrive at this number ( otherwise someone will see "4GiB", recognize not-matching bytes and "fix it" )

```suggestion
    // However, this would allow for only 5 shards per Filecoin sector (if all
    // 5GB). Slightly under 4GB would allow a minimum 7 shards per sector.
    // ( Filecoin data is addressed in 127 byte increments with offsets
    // thus the unusual 4GiB/128*127 value ) 
    maxUploadSize: 4_261_412_864, 
```